# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ppp/hotfix.patch.avr32
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ppp-ppp-2.5.0/pppd/main.c.vanilla	2023-11-30 12:02:03.625367189 +0100
+++ ppp-ppp-2.5.0/pppd/main.c	2023-11-30 12:02:42.677369708 +0100
@@ -1514,10 +1514,10 @@
 	kill_my_pg(sig);
     notify(sigreceived, sig);
     if (waiting) {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-result"
+//#pragma GCC diagnostic push
+//#pragma GCC diagnostic ignored "-Wunused-result"
 	write(sigpipe[1], &sig, sizeof(sig));
-#pragma GCC diagnostic pop
+//#pragma GCC diagnostic pop
     }
 }
 
@@ -1538,10 +1538,10 @@
 	kill_my_pg(sig);
     notify(sigreceived, sig);
     if (waiting) {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-result"
+//#pragma GCC diagnostic push
+//#pragma GCC diagnostic ignored "-Wunused-result"
 	write(sigpipe[1], &sig, sizeof(sig));
-#pragma GCC diagnostic pop
+//#pragma GCC diagnostic pop
     }
 }
 
@@ -1555,10 +1555,10 @@
 {
     got_sigchld = 1;
     if (waiting) {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-result"
+//#pragma GCC diagnostic push
+//#pragma GCC diagnostic ignored "-Wunused-result"
 	write(sigpipe[1], &sig, sizeof(sig));
-#pragma GCC diagnostic pop
+//#pragma GCC diagnostic pop
     }
 }
 
@@ -1592,10 +1592,10 @@
 {
     got_sigusr2 = 1;
     if (waiting) {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-result"
+//#pragma GCC diagnostic push
+//#pragma GCC diagnostic ignored "-Wunused-result"
 	write(sigpipe[1], &sig, sizeof(sig));
-#pragma GCC diagnostic pop
+//#pragma GCC diagnostic pop
     }
 }
 
